automation/tests/authentication.spec.ts

```diff
@@ -1,17 +1,150 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, } from '@playwright/test';
+import { AuthenticationPage } from '../pages/authentication.page.ts';
+import dotenv from 'dotenv';
+dotenv.config();
+import { loginCredentials } from '../test-data/authentication_page_data.js';
+import { EmailAuthCredential } from 'firebase/auth/web-extension';
+import { registrationData} from '../test-data/authentication_page_data.js';
+import { randomString } from '../utils/services/creatingData.ts';
+import { randomEmail } from '../utils/services/creatingData.ts';
+
+let authenticationPage: AuthenticationPage;
+
+
 
 test.beforeEach(async({page})=>{
-  await page.goto('http://localhost:5173/');
+  await page.goto('/');
+   authenticationPage = new AuthenticationPage(page);
+});
+
+test.describe('@VB-31 - Navigation to/from login page, and navigation to registration form', ()=> {
+  test('@Smoke - Login Button redirects to login page', async ({page})=> {
```

Comment:
Usually test names should be more human than computer language based. E.g. instead of 'Login Button redirects to login page' you could use 'Go to login page from the home page' and instead of 'Exit Button redirects to guest page' you could use 'Exit login page' and so on.